src/EnergyPlus/UnitarySystem.cc

```diff
@@ -656,6 +656,7 @@ namespace UnitarySystems {
                             Fans::GetFanDesignVolumeFlowRate(state, blankString, blankString, errorsFound, this->m_FanIndex);
                     }
                 }
+                if (this->m_ActualFanVolFlowRate > 0.0) this->m_MyFanFlag = false;
```

Comment:
I didn't read the PR contents in great detail yet so I'm not sure what this change does yet.

Reply:
The else part of the if block never set the trigger flag false. In the debugger I kept stepping through here when it is meant to be a one-shot. But maybe I did not step through here enough since once the fan does have a valid flow rate then maybe it should step back through again using the if part of this if/else which will set MyFanFlag = false. I'll check that and revert this if necessary.

Reply:
@Myoldmopar I reverted this, commented to avoid doing it again, and added unit test to highlight the flow of code.